### PR TITLE
Improve Traditional Chinese translation

### PR DESCRIPTION
**Pull request by muan**

### CODE REVIEW

**Comment by ykzts** [comment]
cc @Artoria2e5 @caasi @shouko @vidarlee

File: config/locales/zh-TW.yml [hunk 1]
```diff
@@ -166,6 +168,15 @@ zh-TW:
     csv: CSV
     follows: 您關注的使用者
     storage: 儲存空間大小
+    mutes: 您靜音的使用者
```
Comment (ykzts) [inline config/locales/zh-TW.yml]: Sort alphabetically.

File: config/locales/zh-TW.yml [hunk 2]
```diff
@@ -231,15 +243,26 @@ zh-TW:
...
     back: 回到 Mastodon
+    development: 開發
     edit_profile: 修改個人資料
     export: 匯出
     import: 匯入
+    notifications: 通知
     preferences: 偏好設定
     settings: 設定
-    two_factor_authentication: 雙因子認證
+    followers: 授權追蹤者
```
Comment (ykzts) [inline config/locales/zh-TW.yml]: Sort alphabetically.

**Review by ykzts — DISMISSED** [review]

File: config/locales/zh-TW.yml [hunk 3]
```diff
@@ -55,7 +55,7 @@ zh-TW:
       perform_full_suspension: 進行停權
       profile_url: 個人檔案網址
       public: 公開
-      push_subscription_expires: PuSH 訂閱逾期
+      push_subscription_expires: 推播訂閱過期
```
Comment (ykzts) [inline config/locales/zh-TW.yml]: Do you need to translate "PuSH"?

**Review by Gargron — APPROVED** [review]

File: config/locales/zh-TW.yml (hunk 3 above)
Comment (Artoria2e5) [reply config/locales/zh-TW.yml]: It usually gets translated as, simply, "push". I guess it's fine to do so.

File: config/locales/zh-TW.yml [hunk 4]
```diff
@@ -231,15 +243,26 @@ zh-TW:
...
+    development: 開發
     edit_profile: 修改個人資料
     export: 匯出
+    followers: 授權追蹤者
     import: 匯入
+    notifications: 通知
     preferences: 偏好設定
     settings: 設定
-    two_factor_authentication: 雙因子認證
+    two_factor_authentication: 兩階段認證
```
Comment (Artoria2e5) [inline config/locales/zh-TW.yml]: Why is everyone turning two factor into "two step" like it's a secruity from obscurity thing??? I \*\*\*\*ing hate whoever defined the terminology.

Comment (caasi) [reply config/locales/zh-TW.yml]: I just realized that Google uses "兩步驟驗證"(two-step) or "雙重驗證"(double?) for this. And LINE uses "二階段認證"(two-step). FYI

Comment (Artoria2e5) [reply config/locales/zh-TW.yml]: yeah those companies have been doing that for English and zh-CN too. still not the best thing to corrupt a translation when the original got it right though.

File: app/javascript/mastodon/locales/zh-TW.json [hunk 5]
```diff
@@ -127,14 +127,14 @@
...
-  "lists.search": "Search among people you follow",
-  "lists.subheading": "Your lists",
+  "lists.account.add": "加到名單裡",
+  "lists.account.remove": "從名單中移除",
+  "lists.delete": "刪除名單",
+  "lists.edit": "修改名單",
+  "lists.new.create": "新增名單",
+  "lists.new.title_placeholder": "名單名稱",
+  "lists.search": "搜尋您關注的使用者",
+  "lists.subheading": "您的名單",
```
Comment (SerCom-KC) [inline app/javascript/mastodon/locales/zh-TW.json]: Twitter's Traditional Chinese uses `列表` for lists. Not sure if it's HK or TW though.

**Pull request merged**